src/sass/_normalize.scss

```diff
@@ -0,0 +1,12 @@
+// Ideally all of this will go in uswds/src/stylesheets/lib/normalize, but until
+//  it does, normalizing css goes in here.
+
+dl {
```

Comment:
This didn't really seem to fit anywhere else, so I put it in a normalize scss file. It should go in uswds probably, but I don't know how to go about doing that, so for now, it's in here.